Add-ons/UmbracoForms/Installation/index-v8.md

```diff
@@ -0,0 +1,13 @@
+# Installation and upgrading
+
+## [Extending Umbraco with the Forms section](Install)
+Extend Umbraco with the form builder section in a few steps  
+
+## [Configuring a license](../../The-Licensing-model)
+Getting Forms out of trial mode
+
```

Comment:
@bergmania 
I noticed the **Keeping Forms up to date** is not on this page.
So, it's correct that you will not be able to upgrade Forms directly from the backoffice in Umbraco 8?

Reply:
@sofietoft, Yes. Thats also mentioned in the PR message :

> * Auto updates from UI is removed, because it should not be available for editors to do the update